src/hooks/useTransactionType.ts

```diff
@@ -55,26 +55,24 @@ export default function useTransactionType({ input, type, weightInput }: Props):
 
   useEffect(() => {
     async function getValues() {
-      let apiPromise, paymentInfo, apiCall;
+      let paymentInfo, apiCall;
```

Comment:
use types on each variable and init them. I think we should always do that as soon as we use TS

Reply:
These types are not easy to express and my understanding was that type script is pretty good in type inference. It's really just used as temporary holders - I'd define them in `switch` branches, but since `switch` is such a poor JS construct they would leak, so the linter prevents me from doing so.

I worked around it, please let me know wdyt.